cmd/auto-complete.go

```diff
@@ -398,6 +398,9 @@ var completeCmds = map[string]complete.Predictor{
 	"/admin/replicate/status": aliasCompleter,
 	"/admin/replicate/remove": aliasCompleter,
 
+	"/admin/cluster/bucket/export": aliasCompleter,
+	"/admin/cluster/bucket/import": aliasCompleter,
```

Comment:
Why isn't this in `mc admin bucket ...`? Since we are dealing with one bucket, I would expect it to be there?

Reply:
it logically belongs with the `mc admin bucket..` commands - however the idea was to move support tools to backup/import  cluster data under `mc admin cluster..`